tests/models/test_models_vae.py

```diff
@@ -302,19 +294,27 @@ def test_tae_tiling(self, in_shape, out_shape):
             dec = model.decode(zeros).sample
             assert dec.shape == out_shape
 
-    def test_stable_diffusion(self):
+    @parameterized.expand([(True,), (False,)])
+    def test_tae_roundtrip(self, enable_tiling):
+        # load the autoencoder
         model = self.get_sd_vae_model()
-        image = self.get_sd_image(seed=33)
+        if enable_tiling:
+            model.enable_tiling()
+
+        # make a black image with a white square in the middle,
+        # which is large enough to split across multiple tiles
+        image = -torch.ones(1, 3, 1024, 1024, device=torch_device)
+        image[..., 256:768, 256:768] = 1.0
 
+        # round-trip the image through the autoencoder
         with torch.no_grad():
             sample = model(image).sample
 
```

Comment:
The test is much appreciated! But this isn't a round-trip, is it?

Reply:
`AutoencoderTiny.forward` runs encode -> quantize -> dequantize -> decode internally https://github.com/madebyollin/diffusers/blob/main/src/diffusers/models/autoencoder_tiny.py#L321, so this test exercises both encoding and decoding

Reply:
It does what now? Oh, so it does, I see. Is that how one trains an autoencoder or something? All good then.